app/src/main/kotlin/com/wire/android/workmanager/worker/MigrationWorker.kt

```diff
@@ -65,6 +68,15 @@ class MigrationWorker
     }
 
     override suspend fun getForegroundInfo(): ForegroundInfo {
+
+        if (notificationChannelsManager.shouldCreateChannel(NotificationConstants.OTHER_CHANNEL_ID)) {
```

Comment:
you don't need to check if channel exists, Android will ignore it if it's already exist